packages/fast-jss-manager-react/package.json

```diff
@@ -21,23 +23,31 @@
     "unit-test": "jest",
     "unit-test:watch": "jest --watch && npm-watch"
   },
-  "license": "MIT",
-  "readme": "README.md",
   "jest": {
+    "collectCoverage": true,
+    "coverageThreshold": {
```

Comment:
why do only some of these have coverage thresholds and why are the ones that do differ from each-other?

Reply:
I added coverage thresholds to ensure we do not slip and get worse. But, ultimately, we need to discuss as a team our code coverage metrics and/or test coverage goals and meet those.